killercoda/step3/foreground.sh

```diff
@@ -0,0 +1 @@
+export KUBECONFIG=/etc/rancher/k3s/k3s.yaml
```

Comment:
do we need to do this on each step? I tried exporting a variable in one step and the moving to another step and the variable was still defined. I think all steps are executed in the same shell session.

Reply:
All steps are executed in the same session, but I was concerned about the user's ability to jump back and forth.

Thinking about it again... it should be fine as long as they don't skip the step, which I don't think it's possible. I'll get rid of the second one.